### ISSUE TITLE
how to find accuracy for train.py ?

### ISSUE THREAD

**Issue by chillaxkrish** [yufengm/Adaptive#17]
please help me how to find accuracy for each epochs ? Is their any code ?

Its showing Cross Entropy Loss value and Perplexity value in the training !!
